addons/mail/static/src/components/discuss/discuss_tests.js

```diff
@@ -2030,33 +2030,36 @@ QUnit.test('new messages separator', async function (assert) {
     };
     await afterNextRender(() => {
         const notifications = [[['my-db', 'mail.channel', 20], data]];
-        this.widget.call('bus_service', 'trigger', 'notification', notifications);
+        this.widget.call('bus_service', 'trigger', 'notification', notifications)
     });
-    assert.strictEqual(
-        document.querySelectorAll(`
-            .o_Discuss_thread .o_ThreadViewer_messageList .o_MessageList_message
-        `).length,
+    assert.containsN(
+        document.body,
+        '.o_Discuss_thread .o_ThreadViewer_messageList .o_MessageList_message',
```

Comment:
As said in original PR, could simplify selectors with single classname in diff, like here just `.o_MessageList_message`.

Reply:
That's weird, I wonder if I didn't fail my rebase as I fixed this :thinking: